mlos_bench/mlos_bench/environments/README.md

```diff
@@ -1,20 +1,32 @@
 # Environments
 
-This directory contains the code for the `Environments` used in the [`mlos_bench`](../../../mlos_bench/) benchmarking automation framework.
+## Overview
+
+This directory contains the code for the [`Environment`](./base_environment.py) classes used in the [`mlos_bench`](../../../mlos_bench/) benchmarking automation framework.
+An [`Environment`](./base_environment.py) is a class that represents a system that can be benchmarked.
+It is responsible for setting up the system, running the benchmark, and tearing down the system.
+Each `Environment` object also keeps track of the current state of the system, and can be used to query the system for metrics.
+
+Environments have [`Tunable`](../tunables/tunable.py) parameters and [`TunableGroups`](../tunables/tunable_groups.py) for controlling their configuration.
+It is generally expected that all [`Tunable`](../tunables/tunable.py) parameters within an `Environment` will have the same cost to change.
```

Comment:
```suggestion
It is generally expected that all [`Tunable`](../tunables/tunable.py) parameters within an `Environment` will have the same cost to change.
> It may therefore make sense to split a portion of the system logically across `Environments` (e.g., boot-time vs. runtime settings).
```